tools/link.py

```diff
@@ -786,6 +786,10 @@ def phase_linker_setup(options, state, newargs):
     # to js.
     settings.WASM = 1
     settings.WASM2JS = 1
+    # Wasm bigint doesn't make sense with wasm2js, since it controls how the
+    # wasm and JS interact.
+    settings.WASM_BIGINT = 0
```

Comment:
Do we error out if `-sWASM=0 -sWASM_BIGINT` are both given on the command line?

Reply:
No, I think this causes us to silently override it. probably we should error if BIGINT is explicit on the command line.